client/src/elements/schema-editor/schema-editor-wrapper.html

```diff
@@ -2,7 +2,8 @@
   <require from="./schema-editor-wrapper.css"></require>
   <div if.bind="getType(schema) === 'object'" class="schema-editor-block">
     <require from="./schema-editor-object"></require>
-    <schema-editor-object
+    <schema-editor-object\
```

Comment:
attention, I don't think that the `\` here is on purpose

Reply:
uh, nice. thanks!